plugin/src/test/java/io/jenkins/plugins/casc/vault/KV1WithUserTest.java

```diff
@@ -0,0 +1,46 @@
+package io.jenkins.plugins.casc.vault;
+
+
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.EnvVarsRule;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import jenkins.model.Jenkins;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.testcontainers.vault.VaultContainer;
+
+import static org.junit.Assert.assertEquals;
+
+// Inspired by https://github.com/BetterCloud/vault-java-driver/blob/master/src/test-integration/java/com/bettercloud/vault/util/VaultContainer.java
+public class KV1WithUserTest {
+
+    @ClassRule
+    public static VaultContainer vaultContainer = VaultTestUtil.createVaultContainer();
+
+    @BeforeClass
+    public static void configureVaultContainer() {
+        VaultTestUtil.configureVaultContainer(vaultContainer);
```

Comment:
You could use `assumeFalse(isWindows())` in the BeforeClass should avoid spinning up the Jenkins instance on Windows.

Reply:
Added it now